core/src/main/java/com/orientechnologies/orient/core/storage/impl/local/paginated/atomicoperations/OAtomicOperationsManager.java

```diff
@@ -29,7 +29,6 @@
 import com.orientechnologies.orient.core.Orient;
```

Comment:
Could you fix observations in this class in separate commit if  you do not mind

Reply:
done